### ISSUE TITLE
BSOD with Brave and Chromium opened at the same time

### ISSUE THREAD

**Issue by Duranis** [brave/browser-laptop#2851]
**Did you search for similar issues before submitting this one?**
Yes - none found

**Describe the issue you encountered:**
Just installed Brave, seemed to be working fine with Firefox opened.  Imported cookies no issues.  Noticed in the "about Brave" window that it used Chromium.  Opened Chromium to see what version it was at, Within ~ a minute BSOD.

**Expected behavior:**
Didn't expect a BSOD from having Chromium browser opened at the same time as Brave.  Also have Firefox still opened as well.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Win 7 x64 Ultimate
- Brave Version:
0.11.1
- Steps to reproduce:  didn't attempt to reproduce it.
    1.
    2.
    3.

- Screenshot if needed:
No screen shot but can provide a memory dump file...unfortunately it is about 1gb in size.  Zipped it is 206mb.

- Any related issues:

**Comment by bbondy** [comment]
I think this is an unfortunate coincidence and not controllable / causable from our code.   Is it reliably reproducible? If not i think this should be closed.

**Comment by Duranis** [comment]
Did just reproduce it.  Was hoping it was a minor fluke but, again with Brave open, I opened Chromium (Firefox not opened this time).  Was on the about screen.  Clicked onto the Settings link within Chromium and BSOD.  Unfortunately, I hadn't changed my minidump yet...here I go again.
Issue may stem more from the Chromium side as it occurred when I clicked the settings sink in Chromium.

**Comment by Duranis** [comment]
Reproduced again.  Minidump was actually changed before didn't realize it.  So captured 2...however, I don't think this is a Brave issue as the BSOD occurs when going from the About info in Chromium to the Settings area.
Granted I haven't used Chromium in some time, installed it, basically replica gui from Chrome - I miss my menu bar... so hadn't used it again.
I uninstalled and will reinstall Chromium to see if it reproduces from there.  If so will uninstall Brave (was going to do that first but I'm more apt to use Brave than Chromium, so...) and uninstall/reinstall Chromium again to see if it reproduces with Brave cleaned from the system.  If not, of course reinstall Brave and then see if it will reproduce - if it does at that point then possibly some conflict in the differences of the 2 x Chromium engines running???
My Chromium version before first uninstall was Chromium Version 51.0.2683.0.

**Comment by Duranis** [comment]
Ok, happy to report that it doesn't seem to be an issue with Brave at all.
Uninstalled/Deleted Chromium and Brave (I use the IObit uninstaller to clean out the apps).
Rebooted to make sure all clean.
Extracted Chromium and repeated same issue without Brave installed.
Needless to say Chromium going into the dumpster - will report the issue to them but I've done enough BSODing for the week.
Brave reinstalled...however, before I did I just curiously launched my Chrome browser and repeated the same issue with it.
Obviously something went awry with Chrome/Chromium or Google itself.  Don't really use Chrome myself much - family does, set as default for them, I only use it for google apps occasionally.
I prefer Firefox at the moment, will see how Brave works out.

Again not a Brave issue - issue with Chrome/Chromium.  Brave just happened to have been the last thing I installed when I ran  into the issue.
